distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Duration.enso

```diff
@@ -65,7 +65,8 @@ ensure_duration object ~action =
 ## Represents the amount of time between two points in time.
 @Builtin_Type
 type Duration
-    ## Create an interval representing the duration between two points in time.
+    ## ICON operators
+       Create an interval representing the duration between two points in time.
```

Comment:
Given that `up_to` and `down_to` on `Date_Range` got the `date_and_time` icon, why shouldn't `Duration.between` be the same?

Reply:
Moved to `date_time`.